libs/components/lookup/src/lib/modules/country-field/country-field.component.ts

```diff
@@ -141,6 +141,13 @@ export class SkyCountryFieldComponent
     return this.#_includePhoneInfo;
   }
 
+  /**
+   * Whether to always show the country field's placeholder text regardless of theme. This is used for the phone field component.
+   * @internal
+   */
+  @Input()
+  public showPlaceholderTextInAllThemes = false;
```

Comment:
Why not just show it for all themes, always?

Reply:
Design does not want to show placeholder text on standalone phone fields in modern theme because the input box label is enough information. But country field is used inside of phone-field's input box where the label is for the phone field and not the country search portion. So we do want to show the placeholder in this one off situation. That is why I marked this as `internal` because it is just for our use. I tried to mention that in the comment on the property.

Reply:
Instead of creating something on our public API, it might be better to create a shared service or injection token that the country field could respond to if it's defined. We'll need a better name, but something like:
**country-field.component.ts**
```
@Optional() context? CONTEXT // <-- phone field would provide this value, otherwise, it's undefined

if (context?.showPlaceholderText) { ... }
```
This is the preferred way for two components to interact, and it would prevent our consumers from using an "internal" API unknowingly.

Reply:
I have created an injection token for this. I also did a tad bit of linting cleanup while I added a couple unit tests to ensure that the new token was working.